app/javascript/article-form/elements/__tests__/__snapshots__/tags.test.jsx.snap

```diff
@@ -100,32 +78,10 @@ Object {
   "prevLen": 0,
   "searchResults": Array [
     Object {
-      "_highlightResult": Object {
-        "bg_color_hex": Object {
-          "matchLevel": "none",
-          "matchedWords": Array [],
-          "value": "#888751",
-        },
-        "name": Object {
-          "fullyHighlighted": false,
-          "matchLevel": "full",
-          "matchedWords": Array [
-            "gi",
-          ],
-          "value": "<em>gi</em>t",
-        },
-        "text_color_hex": Object {
-          "matchLevel": "none",
-          "matchedWords": Array [],
-          "value": "#56c938",
-        },
-      },
-      "bg_color_hex": "#888751",
```

Comment:
It probably doesn't matter for the use case in this PR, but if other parts of the application start to use Elastic Search for tag search, I imagine we'll need this other metadata about tags? e.g. `bg_color_hex`, `text_color_hex`.

Reply:
There is a possibility we end up needing other attributes in the future but for now I kept it only to what we are using. One thing about Elasticsearch is that you can very easily add fields, you CANNOT take them away without creating an entirely new index and reindexing all of your data. For this reason, I like to move slow on adding things and only do it when we absolutely need to.